chaco/plot_containers.py

```diff
@@ -19,27 +19,25 @@
     Array,
     Either,
     Enum,
-    Float,
     Instance,
     List,
     Property,
-    String,
+    Str,
     Trait,
     Tuple,
     Int,
 )
-from enable.simple_layout import (
-    simple_container_get_preferred_size,
-    simple_container_do_layout,
-)
+from enable.api import Container, OverlayContainer
+from enable.stacked_container import HStackedContainer, VStackedContainer
```

Comment:
`enable` needs to expose these via the api module.

Reply:
ref: https://github.com/enthought/enable/pull/779